src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,39 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+public class AddDriverToCarController extends HttpServlet {
+    private static final String PACKAGE = "mate";
+    private static final String PATH_TO_JSP = "/WEB-INF/views/cars/drivers/addDriversToCar.jsp";
+    private static final Injector injector = Injector.getInstance(PACKAGE);
+    private final CarService carService = (CarService) injector.getInstance(CarService.class);
+    private final DriverService driverService = (DriverService) injector
+            .getInstance(DriverService.class);
+
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher(PATH_TO_JSP).forward(req, resp);
+    }
+
+    @Override
+    public void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long driverId = Long.valueOf(req.getParameter("driver_id"));
+        Long carId = Long.valueOf(req.getParameter("car_id"));
+        Car car = carService.get(carId);
+        Driver driver = driverService.get(driverId);
+        car.getDrivers().add(driver);
+        carService.update(car);
+        resp.sendRedirect(req.getContextPath() + "/index");
```

Comment:
is /index page the most appropriate one here?

Reply:
"HINT: while implementing the last requirement, create just two fields on JSP where you will pass corresponding id values. You can redirect to /index page or any other page after your business logic in doPost() method is applied."
I took it from the task hint, but maybe I got it wrong?